src/base/forms/_base.scss

```diff
@@ -12,12 +12,16 @@ label,
 legend,
 .checkbox {
 	&.required {
-		&:before {
-			@extend %forms-color-red-bold;
+		/* Set required asterisk styles... unless any parent element uses a disable class */
+		/* Credit: Stack Overflow answer (https://stackoverflow.com/a/73773370) by magda and Zach Jensz (zach-jensz) */
+		&:not(.required-no-asterisk .required) {
```

Comment:
Should a ``:where()`` pseudo-class be leveraged to reduce this selector's specificity?

What I had in mind was backwards-compatibility with any pre-existing WET implementations that might've tried overriding the required asterisk styles (would be very surprised if anyone's bothered to try :P).

Reply:
Was determined that it'd be best not to proceed with this.